llm_on_ray/common/trainer/default_trainer.py

```diff
@@ -219,9 +219,9 @@ def train(self):
                                 "train_epoch": idx,
                                 "total_epochs": num_train_epochs,
                                 "train_step": step,
-                                "total_steps": min(max_train_step, total_steps)
+                                "steps_per_epoch": min(max_train_step, steps_per_epoch)
```

Comment:
```suggestion
                                "total_steps": min(max_train_step, steps_per_epoch * num_train_epochs)
```

Reply:
clarified variable `max_train_steps` meaning, and updated its related logic.